File: Backend-Java/src/main/java/org/example/Infrastructure/dto/commentDTO/CommentCreateDTO.java
package org.example.Infrastructure.dto.commentDTO;

import lombok.Data;
import org.springframework.web.multipart.MultipartFile;

import java.util.List;

@Data
public class CommentCreateDTO {
    private String title;
    private String message;
    private int stars;
    private int userId;
    private Integer productId;
    private List<MultipartFile> images;
}

File: Backend-C#/Infrastructure.MyShop/Models/DTO/CommentDTO/CommentCreateDTO.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Infrastructure.MyShop.Models.DTO.CommentDTO
{
    public class CommentCreateDTO
    {
        public string Title { get; set; }
        public string Message { get; set; }
        public int Stars { get; set; }
        public int UserId { get; set; }
        public int? ProductId { get; set; }
        public List<IFormFile>? Images { get; set; }
    }
}
